composition/string_fragment.go

```diff
@@ -5,13 +5,29 @@ import (
 )
 
 // StringFragment is a simple template based representation of a fragment.
-type StringFragment string
+type StringFragment struct {
```

Comment:
@cborgolte I have to correct myself. In our discussion earlier i thought about methods, not attributes. So if you want to add attributes for the CSS-deduplication later on i'm fine with it.

If you just want to add methods in the future, let it be a string